.github/workflows/ci.yaml

```diff
@@ -5,14 +5,22 @@ on:
     - cron:  '0 0 * * *'
 jobs:
   test:
-    name: PHPUnit (PHP ${{ matrix.php-versions }}) (${{ matrix.composer }} dependencies)
-    runs-on: ${{ matrix.operating-system }}
+    name: PHPUnit (PHP ${{ matrix.php-versions }}) (SF ${{ matrix.sf-version }}) (${{ matrix.composer }} dependencies)
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        operating-system: [ubuntu-latest]
+        sf-version: ["*"]
         php-versions: ['8.1', '8.2', '8.3', '8.4']
-        composer: ["lowest", "highest"]
+        composer: [ "lowest", "highest" ]
+        include:
+          - sf-version: "5.4.*"
```

Comment:
TIL the `include` feature in gh actions matrix, thanks :D

Reply:
It is really handy for open source bundles like this one.